src/components/SectionEditModal/helpers.spec.js

```diff
@@ -47,7 +47,8 @@ describe('createGeojsonWithModifiedMode', () => {
 
     expect(modifiedGeojson.series[1].features[1].features[1]).toMatchObject({
       properties: {
-        sensed_mode: 'PredictedModeTypes.BUS'
+        sensed_mode: 'PredictedModeTypes.WALKING',
+        manual_mode: 'PredictedModeTypes.BUS'
```

Comment:
The `PredictedModeTypes` sounds contradictory with the `manual_mode` ; we might just have `manual_mode: BUS` and adapt the `getFeatureMode` to do the split for `sensed_mode`, but not `manual_mode`?